src/Typeahead.vue

```diff
@@ -11,7 +11,7 @@
       @keydown.esc="reset"
       @blur="showDropdown = false"
     />
-    <ul :class="{'show':showDropdown}" class="dropdown-menu search-dropdown-menu" ref="dropdown">
+    <ul :class="dropdownMenuClasses" class="dropdown-menu search-dropdown-menu" ref="dropdown">
```

Comment:
since `dropdownMenuClasses` is computed (and cached), we can add `"dropdown-menu search-dropdown-menu"` to the array too.